roles/sap_ha_install_anydb_ibmdb2/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  namespace: community
+  author: Sean Freeman
+  description: HA for SAP AnyDB - IBM Db2
+  company: Red Hat, Inc.
+  license: Apache-2.0
+  min_ansible_version: 2.9
```

Comment:
which is it 2.9 or 2.12 as in runtime.yml?

Reply:
Copied template from all other Roles, out of scope for PR